reference/latest/src/main/java/com/example/docs/FabricDocsReference.java

```diff
@@ -34,6 +36,7 @@ public void onInitialize() {
 
 		LOGGER.info("Hello Fabric world!");
 		//#entrypoint
+		ModEnchantmentEffects.registerModEnchantmentEffects();
```

Comment:
Just make a new entrypoint for enchantments, it prevents mess. Dont reuse existing entrypoints.

The particle stuff will be moved to a seperate entrypoint at a later date.